src/SonarScanner.MSBuild.Common/AnalysisProperties/AggregatePropertiesProvider.cs

```diff
@@ -32,15 +32,17 @@ namespace SonarScanner.MSBuild.Common;
 public class AggregatePropertiesProvider : IAnalysisPropertyProvider
 {
     /// <summary>
-    /// Ordered list of child providers
+    /// Ordered list of child providers.
     /// </summary>
-    private readonly IAnalysisPropertyProvider[] providers;
+    public IAnalysisPropertyProvider[] Providers { get; }
```

Comment:
This one should not be made public.
```suggestion
    private readonly IAnalysisPropertyProvider[] providers;
```